src/main/java/frc/robot/subsystems/climber/Climber.java

```diff
@@ -0,0 +1,125 @@
+package frc.robot.subsystems.climber;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.can.TalonFX;
+import edu.wpi.first.wpilibj.Solenoid;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+import frc.robot.Ports;
+import frc.robot.subsystems.UnitModel;
+
+public class Climber extends SubsystemBase {
+    private final TalonFX master = new TalonFX(Ports.Climber.MASTER);
+    private final TalonFX slave = new TalonFX(Ports.Climber.SLAVE);
+    private final UnitModel unitModel = new UnitModel(Constants.Climber.TICKS_PER_METER);
+    private final Solenoid stopper = new Solenoid(Ports.Climber.STOPPER);
+    private final Solenoid gearboxPiston = new Solenoid(Ports.Climber.GEARBOX_PISTON);
+
+    public Climber() {
+        slave.follow(master);
+
+        master.setInverted(Ports.Climber.MASTER_INVERTED);
+        slave.setInverted(Ports.Climber.SLAVE_INVERTED);
+
+        master.setSensorPhase(Ports.Climber.MASTER_SENSOR_PHASE_INVERTED);
+        slave.setSensorPhase(Ports.Climber.SLAVE_SENSOR_PHASE_INVERTED);
+
+        master.config_kP(0, Constants.Climber.kP);
+        master.config_kI(0, Constants.Climber.kI);
+        master.config_kD(0, Constants.Climber.kD);
+
+        slave.config_kP(0, Constants.Climber.kP);
+        slave.config_kI(0, Constants.Climber.kI);
+        slave.config_kD(0, Constants.Climber.kD);
+    }
+
+    /**
+     * Get the climber's height.
+     *
+     * @return the climber's height [double].

```

Comment:
```suggestion
     * @return the climber's height [m].
```
You don't need the `[double]`. You can include the units, but the return type is written in the [function's signature](https://en.m.wikipedia.org/wiki/Type_signature).